test/recipes/95-test_external_tlsfuzzer_data/cert.json.in

```diff
@@ -50,5 +50,18 @@
         "arguments" : ["-p", "@PORT@",
                        "-d"]}
      ]
+    },
+    {"server_command": ["@SERVER@", "s_server", "-www",
+                 "-key", "tests/serverX509Key.pem",
+                 "-cert", "tests/serverX509Cert.pem"],
+     "environment": {"PYTHONPATH" : "."},
+     "server_hostname": "localhost",
+     "server_port": @PORT@,
+     "tests" : [
+        {"name" : "test-invalid-client-hello-w-record-overflow.py",
```

Comment:
this can be added to the list on line 46, we don't need to start a new server for this script